bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        output_format='regex',
+        output_regex=r"Error in '(?P<filename>.+)': "
```

Comment:
Also this only works if all the output's strictly comply to the format of regex.So make sure most of the output's are in this above format.